config/src/main/resources/experimental.json

```diff
@@ -35,5 +35,6 @@
       "comment": "private key and this comment are ignored.  In a real chain, the private key should NOT be stored",
       "balance": "90000000000000000000000"
     }
-  }
+  },
+  "depositContractAddress": "0x00000000219ab540356cbb839cbe05303d7705fa"
```

Comment:
This should be within the `config` block at the top, where you have it here makes it part of the genesis block.